### DISCUSSION TITLE
Break line when sending message

### DISCUSSION THREAD

**Discussion by rv5gestao** [gabrieldwight/Whatsapp-Business-Cloud-Api-Net#33] [General]
How include break line for send a message in body?

**Comment by gabrieldwight** [comment]
Does it work with `\n` or `Environment.newline` when you add to the text message with string interpolation?

**Comment by rv5gestao** [comment]
Hi Gabriel,

Yes, now is working. The problem is when insert \n in example of sending,
this is transformed in \\n.



Atenciosamente,

Carlos França

Co-Founder

RV5 Tech
3195832296
***@***.***
www.rv5.com.br


Em sex., 17 de fev. de 2023 às 18:29, Gabriel Odero <
***@***.***> escreveu:

**Reply by theCJMan** [reply]
I have added Multiline support to Text messages, I will make a pull request soon again back to the main project
Basically replacing any \n and \\n with Environment.newline, but we also double check for \r\n else we might end with \r\r\n...
